src/libecalc/dto/result/emission.py

```diff
@@ -65,6 +67,26 @@ def create_empty(cls, name: str, timesteps: List[datetime]):
         )
 
 
+class PartialEmissionResult(TabularTimeSeries):
```

Comment:
We might want to have a temporary dto result, before calculating cumulatives....maybe we can remove it and calculate cumulatives right away